src/utils/index.js

```diff
@@ -1,6 +1,21 @@
-/* eslint-disable no-restricted-globals */
-/* eslint-disable radix */
 /* eslint-disable no-plusplus */
+/* eslint-disable radix */
+/* eslint-disable no-confusing-arrow */
+/* eslint-disable no-restricted-globals */
+import axios from 'axios';
+
+export const apiInstance = axios.create({
+  baseURL: 'https://indiana-ah-staging.herokuapp.com/api/v1/',
```

Comment:
Why are we not pointing to 'https://indiana-ah-master.herokuapp.com/api/v1'?

Reply:
This is what I used to test my functionality for now since the master branch (backend) isn't ready yet.